py/server/deephaven/table_factory.py

```diff
@@ -47,13 +47,14 @@ def empty_table(size: int) -> Table:
         raise DHError(e, "failed to create an empty table.") from e
 
 
-def time_table(period: Union[str, int], start_time: str = None) -> Table:
+def time_table(period: Union[str, int], start_time: str = None, blink_table: bool = None) -> Table:
     """Creates a table that adds a new row on a regular interval.
 
     Args:
         period (Union[str, int]): time interval between new row additions, can be expressed as an integer in
             nanoseconds or a time interval string, e.g. "PT00:00:00.001"
-        start_time (str): start time for adding new rows
+        start_time (str, optional): start time for adding new rows, defaults to None
+        blink_table (bool, optional): if the time table should be a blink table, defaults to None
```

Comment:
1. should we improve the start_time docstring, explicitly state what the default value 'None' means?
2. since blink_table is a bool, it might be better to default it to False, a bit more explicit and clear